src/makielayout/layoutables/colorbar.jl

```diff
@@ -58,7 +58,6 @@ end
 
 function layoutable(::Type{<:Colorbar}, fig_or_scene; bbox = nothing, kwargs...)
     topscene = get_topscene(fig_or_scene)
-    attrs = merge!(Attributes(kwargs), default_attributes(Colorbar, topscene).attributes)
```

Comment:
Unrelated, but it seems like this line doesn't do anything and was probably forgotten in a previous PR.